### DISCUSSION TITLE
Very slow sync on mobile

### DISCUSSION THREAD

**Discussion by Icerainman** [remotely-save/remotely-save#741] [Q&A]
Thanks for the plugin, it works very well on PC/MAC, while it take very long time on mobile side. So appreciate if anyone could advice how to improve it. Thanks

**Comment by fyears** [comment]
how many files do you have? are u using s3?

**Reply by Icerainman** [reply]
Thanks for your reply. I have more than 3000 notes,  so files should be more with attachments. I am using OneDrive to sync. Mac takes 10s to sync , and iPhone takes more than 30s.

**Reply by Icerainman** [reply]
Seems it sucks in step 3-4 
Thanks

**Reply by fyears** [reply]
r u using encryption?

**Reply by Icerainman** [reply]
yes

**Reply by fyears** [reply]
ok that's the reason. encrypting and decrypting are cpu-heavy operations. mobile device suffers from the performance deficiencies. no easy way to work around, sorry

**Reply by Icerainman** [reply]
Ok, well noted. Thanks

**Reply by Icerainman** [reply]
Thanks for your help and plugin again. I will try

**Comment by neuromaancer** [comment]
Hi, I am suffering the same in iOS. How can I remove the encryption? I am using S3. @Icerainman
